jobs/JGLOBAL_ARCHIVE

```diff
@@ -9,21 +9,45 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "arch" -c "base arch"
 ##############################################
 export CDUMP=${RUN/enkf}
 
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS COM_ATMOS_BUFR COM_ATMOS_GEMPAK \
-  COM_ATMOS_GENESIS COM_ATMOS_HISTORY COM_ATMOS_INPUT COM_ATMOS_MASTER COM_ATMOS_RESTART \
-  COM_ATMOS_TRACK COM_ATMOS_WMO \
-  COM_CHEM_HISTORY COM_CHEM_ANALYSIS\
-  COM_MED_RESTART \
-  COM_SNOW_ANALYSIS \
-  COM_ICE_HISTORY COM_ICE_INPUT COM_ICE_RESTART COM_ICE_GRIB \
-  COM_OBS COM_TOP \
-  COM_OCEAN_HISTORY COM_OCEAN_RESTART COM_OCEAN_GRIB COM_OCEAN_NETCDF \
-  COM_OCEAN_ANALYSIS \
-  COM_WAVE_GRID COM_WAVE_HISTORY COM_WAVE_STATION COM_WAVE_RESTART \
-  COM_ATMOS_OZNMON COM_ATMOS_RADMON COM_ATMOS_MINMON COM_CONF
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
```

Comment:
Please use only spaces to indent. Mixing in tabs is one of the reasons you keep having indentation errors.